integration-testing/http_client_test.py

```diff
@@ -77,10 +77,27 @@ def test_simple_post(self):
     def test_simple_download(self):
```

Comment:
let's make the existing tests be explicitly "_h1" tests
modify the test names and pass "--http1.1"